native/CMakeLists.txt

```diff
@@ -581,6 +597,20 @@ cocos_source_files(
     cocos/platform/interfaces/modules/XRCommon.h
     cocos/platform/interfaces/modules/IXRInterface.h
 )
+elseif(OPENHARMONY)
+    cocos_source_files(
+        cocos/platform/interfaces/modules/ISystem.h
+        cocos/platform/interfaces/modules/ISystem.cpp
+        cocos/platform/interfaces/modules/ISystemWindow.h
+        cocos/platform/openharmony/modules/System.cpp
+        cocos/platform/openharmony/modules/SystemWindow.cpp
+        cocos/platform/openharmony/modules/SystemWindowManager.h
+        cocos/platform/openharmony/modules/SystemWindowManager.cpp
+        cocos/platform/interfaces/modules/Device.cpp
+        cocos/platform/interfaces/modules/Device.h
+        cocos/platform/interfaces/modules/IScreen.h
+    )
+endif()
```

Comment:
Why OH should use different header files?

Reply:
It should be a very early adaptation legacy issue